InstrumentDB/3 Reports.sql

```diff
@@ -0,0 +1,35 @@
+use InstrumentDB 
+go
+
+/*
+1) I need to know if I have any repeat customers and what they rented
+2)I need to know the most popular category of instrument that has ever been rented
+3) I need to know which instrument (that is not being currently rented) made the most profit
+4) I need to know which instruments were rented for only one month (so I could know if I should just sell
+them off)
+*/
+
+--1
+select i.CustomerName, i.InstrumentName
+from Instrument I
+group by i.CustomerName, i.InstrumentName
+having COUNT(i.CustomerName)>2
+
+--2
```

Comment:
-- LL This question should be looking at the category of the instrument instead of the name